docs/blueprint-landing.css

```diff
@@ -1067,7 +1069,9 @@ span.pt-icon::before {
     outline: none !important; }
 
 /** Copyright 2015 Palantir Technologies, Inc. All rights reserved.
-// Licensed under the Apache License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0 */
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS */
```

Comment:
let's please fix the comment format. either all `/** ... */` block comments or `//` on every line. i prefer the former as it's what we used to do.